docs/using-oci-flexvolume-driver.md

```diff
@@ -0,0 +1,140 @@
+# OCI Flexvolume Driver
+
+The Oracle Cloud Infrastructure (OCI) Flexvolume driver enables attaching of
+[OCI Block Storage][1] volumes to Kubernetes Nodes and mounting these volumes
+into Pods via the [Flexvolume][2] plugin interface.
+
+We recommend you use this driver in conjunction with the OCI Volume Provisioner.
+See the [oci-volume-provisioner][3] for more information.
+
+## Deployment
+
+#### Configuration
+
+You will first need to create a cloud-provider configuration file tailored to
+your OCI account and Kubernetes cluster. Please see the provider configuration
+documentation [here][5].
+
+```bash
+$ kubectl create secret generic oci-volume-provisioner \
+  -n kube-system \
+  --from-file=config.yaml=cloud-provider.yaml
+```
+
+### Kubernetes API Access
+
+The flexvolume driver requires access to the Kubernetes API to resolve Node
+names to the corosponding Instance OCID. A kubeconfig file with appropriate
+permissions (RBAC: nodes/get) needs to be provided via a Kubernetes Secret
```

Comment:
Would it be worth putting a tiny RBAC example off to the side in a link? I know it is not too hard, but, if people are new and just want to get it all running it might stop them adding global access. Up to you.